packages/drivers/odsp-driver/src/test/odspCreateContainer.spec.ts

```diff
@@ -140,12 +140,14 @@ describe("Odsp Create Container Test", () => {
 				[
 					// Due to retry logic in getWithRetryForTokenRefresh() for OdspErrorTypes.incorrectServerResponse
 					// Need to mock two calls
-					async () => okResponse({}, {}),
-					async () => okResponse({}, {}),
+					async (): Promise<object> => okResponse({}, {}),
+					async (): Promise<object> => okResponse({}, {}),
 				],
 			);
-		} catch (error: any) {
-			assert.strictEqual(error.statusCode, undefined, "Wrong error code");
+		} catch (error: unknown) {
+			assert(isFluidError(error), "Error should be IFluidError");
```

Comment:
Any particular reason for the new check?

Reply:
It does the type narrowing to prevent needing to cast to any (or whatever) in subsequent lines.